arboretum/kubernetes/README.md

```diff
@@ -17,7 +17,76 @@ to include the fetchers and checks from this library in your downstream project.
 
 ## Fetchers
 
-Fetchers coming soon...
+### Cluster Resource
+
+* Class: [ClusterResourceFetcher][fetch-cluster-resource]
+* Purpose: Write the resources of **stand-alone** Kubernetes clusters to the
+evidence locker.  **NOTE:** Do not use this fetcher for managed clusters.
+Instead use the [IBM Cloud cluster list fetcher][ibm-cloud-cluster-list-fetcher].
+* Behavior: Retrieve stand-alone Kubernetes cluster resource data for the provided
+list of clusters.  TTL is set to 1 day.
+* Configuration elements:
+  * `org.kubernetes.cluster_resources.clusters`
+    * Required
+    * List of dictionaries:
+      * `label`: An arbitrary label to specify token in credentials file
+      * `server` A URL as a string to a Kubernetes stand-alone cluster
+  * `org.kubernetes.cluster_resources.types`
+    * Optional
+    * List of resource types as strings
+      * NOTE: For core group API resources, the resource name must be in
+      _plural form_.  See example below.
+      * NOTE: For other "custom" API resources, the resource name must be in the
+      following format: `"APIGROUP/VERSION/NAME"`.  See example below.
+    * Use if looking to override the default of `["nodes", "pods", "configmaps"]`.
+    Otherwise do not include.
+* Expected configuration:
+
+  ```json
+  {
+    "org": {
+      "kubernetes": {
+        "cluster_resources": {
+          "clusters": [
+            {
+              "label": "devcluster1",
+              "server": "https://myserver1:30000"
+            }
+          ],
+          "types": ["nodes", "pods", "apigroup.example.com/v1/mycustom"]
```

Comment:
Is your intent to apply the `cluster_resources.types` to all `cluster_resources.clusters` supplied?  Would there ever be a need to allow for different types per cluster?

Reply:
I talked about this with Simon and as a result we decided as is.  As far as I remember, the intent is simplification.  This fetcher does not throw exception even if a type is not defined in a cluster (ignoring the error), and therefore a user can specify superset to this config parameter (e.g., if attempting fetching `"pod"` from cluster A and fetching `"mycustom"` custom resource from cluster B, a user should specify `["pod","mycustom']` here, and ignore "pod" of cluster B and ignore "mycustom" of cluster A.